### DISCUSSION TITLE
I found some documentation discrepancies

### DISCUSSION THREAD

**Discussion by ovizii** [garethgeorge/backrest#112] [Ideas]
I found some differences between:
https://hub.docker.com/r/garethgeorge/backrest and https://github.com/garethgeorge/backrest 

1. the compose samples differ.
2. the variables differ. 

i.e. one place mentions: 
`BACKREST_RESTIC_BIN - the path to the restic binary. Defaults managed version of restic which will be downloaded and installed in the data directory.`

the other one says:
`BACKREST_RESTIC_COMMAND - the path to the restic binary. Defaults managed version of restic which will be downloaded and installed in the data directory`

Please don't take this as a complaint. I just wanted to point it out as it will confuse new users.

**Comment by garethgeorge** [comment]
Thanks for calling this out! Just went ahead and updated the env variable docs on docker hub. These should now match (and as I'm sure you found BACKREST_RESTIC_COMMAND is the correct one )
